lib/reports/ppl-list/parse.js

```diff
@@ -56,7 +56,8 @@ const parse = project => {
     establishment: project.name,
     nhps: hasSpecies(project, nhps) ? 'yes' : 'no',
     catsOrDogs: hasSpecies(project, catsOrDogs) ? 'yes' : 'no',
-    equidae: hasSpecies(project, equidae) ? 'yes' : 'no'
+    equidae: hasSpecies(project, equidae) ? 'yes' : 'no',
+    raDate: project.ra_date
```

Comment:
```suggestion
    raDate: project.ra_date ? moment(project.ra_date).format('YYYY-MM-DD') : ''
```

Per `revocation_date` above.